src/patternfly/components/Tabs/tabs.scss

```diff
@@ -1,104 +1,115 @@
+$pf-c-tabs--breakpoint-map: build-breakpoint-map("base", "md", "lg", "xl", "2xl");
+$pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
+
 .pf-c-tabs {
+  --pf-c-tabs--Inset: 0;
+  --pf-c-tabs--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-tabs--before--BorderColor: var(--pf-global--BorderColor--dark-100);
+
   // Primary tab items
   --pf-c-tabs__item--BackgroundColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-tabs__item--BorderColor: var(--pf-global--BorderColor--100);
   --pf-c-tabs__item--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-tabs__item--m-current--ZIndex: var(--pf-global--ZIndex--sm);
-  --pf-c-tabs__item--m-current--Color: var(--pf-global--active-color--100);
-  --pf-c-tabs__item--m-current--BorderTopWidth: var(--pf-global--BorderWidth--md);
-
-  // Secondary tab items
-  --pf-c-tabs__item--hover--Color: var(--pf-global--Color--dark-200);
+  --pf-c-tabs__item--m-current--Color: var(--pf-global--Color--100);
 
   // Tab buttons
-  --pf-c-tabs__button--Color: var(--pf-global--Color--100);
+  --pf-c-tabs__button--Color: var(--pf-global--Color--200);
   --pf-c-tabs__button--FontWeight: var(--pf-global--FontWeight--normal);
   --pf-c-tabs__button--Background: transparent;
   --pf-c-tabs__button--OutlineOffset: calc(-1 * var(--pf-global--spacer--xs));
   --pf-c-tabs__button--PaddingTop: var(--pf-global--spacer--sm);
-  --pf-c-tabs__button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-tabs__button--PaddingBottom: var(--pf-global--spacer--sm);
-  --pf-c-tabs__button--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-tabs__button--md--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-tabs__button--md--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-tabs__button--md--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-tabs__button--md--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-tabs__button--BorderStyle: solid;
+  --pf-c-tabs--m-vertical--c-tabs__button--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-tabs--m-vertical--c-tabs__button--PaddingRight: var(--pf-global--spacer--md);
```

Comment:
I also wonder if you need the padding right and left vars here, since they're both medium size